lib/api.js

```diff
@@ -109,27 +109,11 @@ var api = {
    * }
    * ```
    */
-  messages: []
-}
+  this.messages = []
 
-/**
```

Comment:
Please don't remove the docs

Reply:
this not need because `Api` is constructor and inheritance occurs when the plugin is called